lib/featureDetection.js

```diff
@@ -32,7 +32,7 @@ function getConfigVarFlag(stdout) {
 }
 
 function getPathConfigFlag(stdout) {
-  if (stdout.indexOf(' --path.conf ') > -1) {
+  if (stdout.indexOf('-Dproperty=value') > -1) {
```

Comment:
Why not check for `--property=value`, since that's actually how we are passing the conf, not using `-Dpath.conf`